docs/python/editing.md

```diff
@@ -177,6 +177,14 @@ You can invoke this command by selecting the lines of code you wish to extract a
 
 ![Refactoring code into a method](images/editing/refactorExtractMethod.gif)
 
+### Rename Module
+
+After a Python file/module is renamed, Pylance can find all instances that need to be updated and provide you with a preview of all the changes.
```

Comment:
maybe "After a Python file/folder is rename, Pylance can find all instances of the module that may need to be updated", or something along those lines? Just to give the idea that it will also work for folders